src/modules/article/dto/received/get-by-category.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class GetByCategoryDto {
+  @ApiProperty({ description: 'category name', example: 'one of the names of the categories' })
```

Comment:
example: 'one of the names of the categories' => if you add example, put example, not description

Reply:
example: 'Love'  or something else

Reply:
done